File: skynda_backend/src/java/me/skynda/common/interfaces/services/IClassificationService.java
package me.skynda.common.interfaces.services;

import me.skynda.classification.dto.ClassificationResponseDto;

import java.util.List;

public interface IClassificationService {
    IList<ClassificationResponseDto> getByType(String param);
    IList<ClassificationResponseDto> getByTypeAndVehicleBound(String param);
}

File: skynda_dotnet_backend/Triven.Domain/Services/IClassificationService.cs
using System.Collections.Generic;
using Triven.Domain.Results;
using Triven.Domain.ViewModels.Classification;

namespace Triven.Domain.Services
{
    public interface IClassificationService
    {
        ServiceResult<IList<ClassificationViewModel>> GetByType(string param);
        ServiceResult<IList<ClassificationViewModel>> GetByTypeAndVehicleBound(string param);
    }
}
